htlcswitch/test_utils.go

```diff
@@ -1056,14 +1056,14 @@ func serverOptionRejectHtlc(alice, bob, carol bool) serverOption {
 // createTwoClusterChannels creates lightning channels which are needed for
 // a 2 hop network cluster to be initialized.
 func createTwoClusterChannels(t *testing.T, aliceToBob,
-	bobToCarol btcutil.Amount) (*testLightningChannel,
+	bobToAlice btcutil.Amount) (*testLightningChannel,
```

Comment:
While we're here, update the godoc? "2 hop network cluster" is a confusing way to refer to two sides of the same channel

Reply:
Updated to godoc *and* the name of the function.